### PR TITLE
Add conformsLevelX functions.

### LINKED ISSUE #24: Support checking level conformance and level is-a-feature-of
To facilitate the work on changes to the edtf-service described at: https://github.com/unt-libraries/django-edtf/issues/15, we will probably need to replace `isLevel0`, `isLevel1`, `isLevel2` with functionality that distinguishes if the date conforms to a specific level or is a feature of a specific level.

### PR DESCRIPTION
**Pull request by madhulika95b**
@somexpert @ldko 
This PR closes #24

### CODE REVIEW

File: edtf_validate/valid_edtf.py [hunk 1]
```diff
@@ -529,7 +529,7 @@ def is_valid_interval(edtf_candidate):
 
 
 def isLevel0(edtf_candidate):
-    """Checks to see if the date is level 0 valid"""
+    """Checks to see if the date is a feature of level 0"""
```
Comment (ldko) [inline edtf_validate/valid_edtf.py:532]: I think we should change "Checks" to "Check" in the docstrings and end with a period for fitting with [PEP 257](https://www.python.org/dev/peps/pep-0257/). Also, what do you think about slightly changing the wording of the "isLevel" functions to further clarify with:
"""Check if the date is of a feature introduced in level 0."""

File: edtf_validate/valid_edtf.py [hunk 2]
```diff
@@ -560,6 +560,34 @@ def isLevel2(edtf_candidate):
     return result
 
 
+def conformsLevel0(edtf_candidate):
+    """Checks to see if the date is level 0 valid"""
+    if isLevel0(edtf_candidate):
+        return True
+    else:
+        return False
```
Comment (ldko) [inline edtf_validate/valid_edtf.py:568]: Would it work for the body to be:
`return isLevel0(edtf_candidate)`

File: edtf_validate/valid_edtf.py [hunk 3]
```diff
@@ -560,6 +560,34 @@ def isLevel2(edtf_candidate):
...
+def conformsLevel2(edtf_candidate):
+    """Checks to see if the date is level 2 valid"""
+    if (
+        isLevel0(edtf_candidate)
+        or isLevel1(edtf_candidate)
+        or isLevel2(edtf_candidate)
+    ):
+        return True
+    else:
+        return False
```
Comment (ldko) [inline edtf_validate/valid_edtf.py:588]: maybe:
`return isLevel0(edtf_candidate) or isLevel1(edtf_candidate) or isLevel2(edtf_candidate)`

File: edtf_validate/valid_edtf.py [hunk 4]
```diff
@@ -560,6 +560,34 @@ def isLevel2(edtf_candidate):
...
+
+
+def conformsLevel1(edtf_candidate):
+    """Checks to see if the date is level 1 valid"""
+    if isLevel0(edtf_candidate) or isLevel1(edtf_candidate):
+        return True
+    else:
+        return False
```
Comment (ldko) [inline edtf_validate/valid_edtf.py:576]: maybe:
`return isLevel0(edtf_candidate) or isLevel1(edtf_candidate)`

File: edtf_validate/valid_edtf.py (hunk 2 above)
Comment (madhulika95b) [reply edtf_validate/valid_edtf.py:568]: oh..yeah. I will change it here and elsewhere too.

File: edtf_validate/valid_edtf.py (hunk 1 above)
Comment (somexpert) [reply edtf_validate/valid_edtf.py]: Sorry, should have chimed in on this earlier, but how do you feel about something like "Check that the date's max-level feature is from level-0"?

**Pull request merged**